### PR TITLE
fix issue #299 - display filters when no product was found

### LINKED ISSUE #299: Filters are not visible when no product is matching
### What I'm trying to achieve
To look for different kind of products that have different set of attributes.

### Steps to reproduce the problem
1. Go to a category;
2. Select filters in a way that it says "No product matching";
3. The filters should no longer be visible, thus we are unable to edit our incorrect filters.

### What I expected to happen
To be able to edit the filters to get new products.

**System information**
Operating system: Mac OS X
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:66.0) Gecko/20100101 Firefox/66.0

### PR DESCRIPTION
**Pull request by aldomonteiro**
I want to merge this change because...

Fixes #299 

### Screenshots

With this correction, when the filters don't match any product, the filters keep open and the user can correct it. The change affects 2 pages:

1) The "Search" page:
image

2) The "Categories" page:
image

### Pull Request Checklist

1. [N/A] All visible strings are translated with proper context.
1. [N/A] All data-formatting is locale-aware (dates, numbers, and so on).
1. [X] The changes are tested.
1. [X ] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

File: src/views/Search/index.tsx [hunk 1]
```diff
@@ -118,13 +118,13 @@ export const SearchView: React.FC<SearchViewProps> = ({
                     return isOnline ? (
                       <Error error={error.message} />
                     ) : (
-                      <OfflinePlaceholder />
-                    );
+                        <OfflinePlaceholder />
+                      );
```
Comment (piotrgrundas) [inline src/views/Search/index.tsx]: I think the style was correct. Use prettier in vscode with default settings.

**Comment by salwator** [comment]
@aldomonteiro could you please check this PR and apply changes needed?

**Comment by aldomonteiro** [comment]
> @aldomonteiro could you please check this PR and apply changes needed?

Done!

**Comment by bogdal** [comment]
Great work @aldomonteiro 👍

**Pull request merged**